tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -849,6 +849,76 @@ test.describe( 'Nested Tabs tests @nested-tabs', () => {
 
 		await cleanup( wpAdmin );
 	} );
+
+	test( 'Test the nested tabs behaviour when using container flex row @latest', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await setup( wpAdmin );
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		await editor.addWidget( 'heading', container );
+
+		const tabsWidgetId = await editor.addWidget( 'nested-tabs', container );
+
+		await editor.addWidget( 'heading', container );
+
+		const contentContainerOne = editor.getPreviewFrame().locator( `.elementor-element-${ tabsWidgetId } .e-n-tabs-content .e-con.e-active` ),
+			contentContainerOneId = await contentContainerOne.getAttribute( 'data-id' ),
+			contentContainerTwo = editor.getPreviewFrame().locator( `.elementor-element-${ tabsWidgetId } .e-n-tabs-content .e-con:not( .e-active )` ).first(),
+			contentContainerTwoId = await contentContainerTwo.getAttribute( 'data-id' ),
+			contentContainerThree = editor.getPreviewFrame().locator( `.elementor-element-${ tabsWidgetId } .e-n-tabs-content .e-con` ).last(),
+			contentContainerThreeId = await contentContainerThree.getAttribute( 'data-id' );
+
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tabs-content .e-con.e-active' );
+
+		// Act.
+		// Add content
+		// Tab 1.
+		await editor.addWidget( 'video', contentContainerOneId );
+
+		// Tab 2.
+		await editor.getPreviewFrame().locator( '.e-normal:not( .e-active )' ).first().click();
```

Comment:
you could also store the locator for the tabButtons where you define all the other `const's` above like this
```
const contentContainerOne = editor.getPreviewFrame().locator( '.e-n-tabs-content .e-con >> nth=0' ),
			tabButtonOne = editor.getPreviewFrame().locator( 'e-n-tabs-heading .e-normal >> nth=0' ),  /* <--- here */
			contentContainerOneId = await contentContainerOne.getAttribute( 'data-id' ),
```
then to click a tab, which you do more than once per tab, then you just call
```suggestion
		// Tab 1.
		await tabButtonOne.click();
```

Reply:
Updated.